src/deadline/client/api/_attachment.py

```diff
@@ -0,0 +1,82 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+import os
+import boto3
+import json
+
+from contextlib import ExitStack
+from typing import Optional, List, Dict
+
+from deadline.job_attachments.asset_manifests.base_manifest import BaseAssetManifest
+from deadline.job_attachments.asset_manifests.decode import decode_manifest
+from deadline.job_attachments.download import download_files_from_manifests
+from deadline.job_attachments.models import JobAttachmentS3Settings, PathMappingRule
+from deadline.job_attachments.progress_tracker import DownloadSummaryStatistics
+from deadline.client.exceptions import NonValidInputError
+
+
+def attachment_download(
+    manifests: List[str],
+    s3_root_path: str,
+    boto3_session: boto3.Session,
+    path_mapping_rules: Optional[str] = None,
```

Comment:
Is this supposed to be a JSON? Can it be an environment variable or a file?

Reply:
Got it, would it make more sense to be "path_mapping_rules_file" ?

I'm thinking about this: 
- If this argument is dual purposed, ie can be an env variable with the mapping JSON as a string, or file path then the current name is good.

- If it will always be a file, then adding `_file` is more descriptive.

- If we only support file today, but later think we will add env variable or other mechanisms (eg s3://) the current name works too.